priv/sites/zotonic_status/modules/mod_zotonic_site_management/support/zotonic_status_addsite.erl

```diff
@@ -175,7 +175,7 @@ addsite_copy_skel(Name, Options, Context) ->
 addsite_compile(Name, Options, Context) ->
     mod_zotonic_site_management:progress(Name, ?__("Force compile all Erlang files ...", Context), Context),
```

Comment:
Line 176 is too long:     mod_zotonic_site_management:progress(Name, ?__("Force compile all Erlang files ...", Context), Context),.